seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -4584,7 +4584,7 @@
         "assert($(\"h1\").hasClass(\"pink-text\"), 'message: Your <code>h1</code> element should have the class <code>pink-text</code>.');",
         "assert($(\"h1\").hasClass(\"blue-text\"), 'message: Your <code>h1</code> element should have the class <code>blue-text</code>.');",
         "assert($(\"h1\").attr(\"id\") === \"orange-text\", 'message: Your <code>h1</code> element should have the id of <code>orange-text</code>.');",
-        "assert(code.match(/<h1[\\s\\S]*?style/gi) && code.match(/<h1[\\s\\S]*?style[\\s\\S]*?color\\s*?:/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
+        "assert(code.match(/<h1[\\s\\S]*[\\s]+style\\s*?=\\s*(\\'|\\\")?\\s*([\\s\\S]*;+|)*\\s*color\\s*?:\\s*?white\\s*;*\\s*(\\'|\\\")?/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
```

Comment:
Me again! We're making good progress on this @dcgit, sorry for the delay. There are still two cases of `[\\s\\S]*` in your code. Translated, that says: "match any character that is a space, as well as any character that is not a space, zero or more times". It effectively matches any character. Both of these instances could be removed. 
The `[\\s]+` at the beginning does not need to be wrapped in a character set, `\\s+` should have the same effect.
@Bouncey commented previously on `([\\s\\S]*;+|)*`, which has not changed. Before or after removing the group I talked about above, I can not understand what the goal of this is. I believe it should work fine without this, but I'd really love to learn what this is meant to do.
Looking at the other tests, it seems that double quotes only need a single backslash in front of them to be escaped. Single quotes don't have to be escaped.
`\\s*?` the question mark here and is unnecessary, as the * already means zero or more. There are three cases of this that should be updated.
With the current expression, the quotes are optional. Technically, this would work in real code, but I wouldn't teach anybody this way. Should this stay optional? @Bouncey